docs/ai/prompt-engineering-in-dot-net.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Prompt engineering in .NET"
+description: "Learn basic GPT prompt engineering concepts for .NET."
+author: catbutler
+ms.author: [your Microsoft alias or a team alias]
+ms.service: [the approved service name]
+ms.topic: concept-article #Don't change.
+ms.date: [mm/dd/yyyy]
+
+#customer intent: As a .NET developer, I want to understand GPT prompt engineering so that I can build more efficient and effective AI chat apps.
+
+---
+
+# Concept - Prompt engineering in .NET
+
+This article covers basic GPT prompt engineering for .NET, including Semantic Kernel and Azure OpenAI. 
+
+GPT models from OpenAI are prompt-based: they respond to user input text (a prompt) with the most likely series of words to follow (a completion). The basic completion in earlier GPT models is text generated in response to prompts. Newer GPT models produce completions in chat format. 
+
+Consider this text generation example:
+ 
+```Prompt
```

Comment:
I copied the block from an existing article and revised it - I'll figured the original use was to make it language-agnostic, but I can just make it specific to C#.

Reply:
Interesting, I know sometimes the language is rendered as the title of the code block. So maybe the intent is just to show the title of "Prompt" and "Completion". I think there are potentially some a11y issues with that though, because I don't think that's an intended use of the language specifier. Could be a question for MS folks

Reply:
I reformatted so it isn't code blocks.